SQL/Cassandra.sublime-syntax

```diff
@@ -0,0 +1,178 @@
+        2: storage.modifier.cql
+    - match: \b(?i:allow\s+filtering)\b
+      scope: keyword.other.dml.cql
+    - match: \b(?i:copy)\b
+      scope: keyword.other.dml.cql
+      push:
+        - copy-from
+        - begin-group-or-pop
+        - table-name
+        - single-identifier-after-whitespace
+
+  statements:
+    - meta_prepend: true
+    - include: use-db
+    - match: \b(?i:source)\b
+      scope: keyword.other.cql
+
+  built-in-type:
+    - meta_prepend: true
+    - match: \b(?i:set|frozen|map|tuple)(<)
+      scope: storage.type.cql
+      captures:
+        1: punctuation.definition.generic.begin.cql
+      set:
+        - end-generic-type
+        - built-in-type
+
+  end-generic-type:
+    - match: \>
+      scope: storage.type.cql punctuation.definition.generic.end.cql
+      pop: 1
+
+  inside-dict-or-set:
+    - match: \}
+      scope: punctuation.section.braces.end.cql
+      pop: 1
+    - match: ':'
+      scope: punctuation.separator.key-value.cql
+    - include: expressions
+    - match: \w+
+      scope: meta.mapping.key.cql string.unquoted.cql
+
+  inside-array:
+    - match: \]
+      scope: punctuation.section.brackets.end.cql
+      pop: 1
+    - include: expressions
+
+  ddl-create-target:
+    - meta_prepend: true
+    - match: \b(?i:with)\b
+      scope: keyword.other.cql
+    - match: \b(?i:clustering)\b
+      scope: keyword.other.cql
+
+  ddl-statements:
+    - meta_prepend: true
+    - match: \b(?i:create\s+type)\b
+      scope: keyword.other.ddl.sql
+      push:
+        - ddl-create-target
+        - ddl-table-creation-columns
+        - create-condition
+
+  ddl-alter-common:
+    - meta_prepend: false
+
+  inside-ddl-table-creation-columns:
+    - meta_scope: meta.group.table-columns.sql
+    - match: \)
+      scope: punctuation.section.group.end.sql
+      pop: 1
+    - match: \b(?i:(primary\s+key))\s*(\()
+      captures:
+        1: storage.modifier.cql
+        2: meta.group.partition-key.cql punctuation.section.group.begin.cql
+      push: partition-key
+    - match: \b(?i:primary\s+key)\b
+      scope: storage.modifier.cql
+      #push: partition-key
+    - include: expressions
+    - include: ddl-creation-column
+
+  partition-key:
+    - meta_content_scope: meta.group.partition-key.cql
+    - match: \)
+      scope: meta.group.partition-key.cql punctuation.section.group.end.cql
+      pop: 1
+    - match: \(
+      scope: punctuation.section.group.begin.cql
+      push: inside-partition-key-group
+    - match: ','
+      scope: punctuation.separator.sequence.cql
+      set: inside-group
+    - include: column-references
+
+  inside-partition-key-group:
+    - match: \)
+      scope: punctuation.section.group.end.cql
+      pop: 1
+    - match: ','
+      scope: punctuation.separator.sequence.cql
+    - include: column-references
+
+  joins:
+    - match: \bjoin\b
+
+  copy-from:
+    - match: \b(?i:from)\b
+      scope: keyword.other.cql
+      pop: 1
+    - match: ''
+      pop: 1
```

Comment:
Due to the change of `begin-group-or-pop` the following change might be required.

```suggestion
    - match: (?=\S)
      pop: 1
```